traveler-ios-ui/TravelerKitUI/ViewControllers/CancellationViewController.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  CancelOrderViewController.swift
+//  TravelerKitUI
+//
+//  Created by Omar Padierna on 2019-06-04.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import UIKit
+import TravelerKit
+
+protocol CancellationViewControllerDelegate: class {
+    func orderCancelSucceed(_ controller: CancellationViewController, didCancel order:Order)
+}
+
+class CancellationViewController: UITableViewController {
+    @IBOutlet weak var totalRefundLabel: UILabel!
+
+    var quote: CancellationQuote?
+    weak var delegate:CancellationViewControllerDelegate?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        guard let quote = quote else {
+            Log ("No quote", data: nil, level: .error)
+            return
+        }
+
+        totalRefundLabel.text = quote.totalRefund.localizedDescription
```

Comment:
opt for not guarding unless you need. use optional chaining

i.e.

`totalRefundLabel.text = quote?.totalRefund.localizedDescription`